packages/p/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@react-email/p",
+  "version": "0.0.1",
+  "description": "A paragraph is a block of text separated by some white space",
```

Maybe we can omit the "A paragraph is.."

```suggestion
  "description": "A block of text separated by some white space",
```

---

good point

---

done!